src/app/components/home-page/home-page.component.scss

```
@@ -1,13 +1,31 @@
 @import "../../../styles/variables.scss";
 
 h1 {
-  font-size: xx-large;
+  font-size: 10vw;
+  line-height: 100%;
+  margin-top: min(5%, 100px);
```

Le titre est peut-être un peu trop grand
```suggestion
  font-size: 7vw;
  line-height: 100%;
  margin-top: min(5%, 100px);
```